src/features/MovieList/styled.js

```diff
@@ -1,25 +1,29 @@
 import styled from "styled-components";
 
 export const Content = styled.section`
-    max-width: 1368px;
-    margin: auto;
+    max-width: 1400px;
+    padding:0 16px;
+    margin: 0 auto;
     display: flex;
     flex-direction: column;
 `;
 
-export const Title = styled.h1`
+export const Title = styled.h2`
     font-family: 'Poppins';
     font-style: normal;
     font-weight: 600;
```

Comment:
I think we're getting most of it by default already, maybe it's not needed? (Unless we want to be explicit)

Reply:
you right, thanks, i fix it